### ISSUE TITLE
Misplaced use trait in User.php

### ISSUE THREAD

**Issue by thelambofgoat** [santigarcor/laratrust#189]
Know it's not a serious bug, however...

After I ran php artisan laratrust:setup I got such picture in User.php

```
class User extends Model implements AuthenticatableContract,
                                    CanResetPasswordContract
    use LaratrustUserTrait;
{

    use Authenticatable, CanResetPassword;

```

Obviously, script shouldn't place trait before class opening bracket even if User class is seriously far from initial.

**Comment by KKSzymanowski** [comment]
Laratrust doesn't handle this, [Traitor](https://github.com/kkszymanowski/traitor) does. I will fix this.

**Comment by KKSzymanowski** [comment]
I released Traitor 0.2.4. Please update and check if the bug is fixed.

**Comment by thelambofgoat** [comment]
I checked. All is ok now. Great!
